### PR TITLE
Update mandala & remove keep-blocks

### PR DESCRIPTION
**Pull request by zjb0807**

### CODE REVIEW

File: docker-compose-example.yml [hunk 1]
```diff
@@ -26,7 +25,7 @@ services:
       POSTGRES_PASSWORD: postgres
 
   subquery-node:
-    image: acala/evm-subql:d9ee626
+    image: acala/evm-subql:0a99a2f
```
Comment (zjb0807) [inline docker-compose-example.yml:28]: need update again after merge

Comment (shunjizhan) [reply docker-compose-example.yml:28]:
```suggestion
    image: acala/evm-subql:0a99a2f
     image: acala/evm-subql:v2.5.22
```

Comment (shunjizhan) [reply docker-compose-example.yml:28]: v2.5.22 will be latest, I am building now

**Review by shunjizhan — APPROVED** [review]

File: docker-compose-example.yml (hunk 1 above)
Comment (zjb0807) [reply docker-compose-example.yml:28]: The `chainId` in `evm-subql/project.yaml` has been changed and needs to be merged and updated again

File: docker-compose-example.yml [hunk 2]
```diff
@@ -73,7 +72,7 @@ services:
       - --indexer=http://subquery-node:3000
 
   eth-rpc-adapter-server:
-    image: acala/eth-rpc-adapter:d9ee626
+    image: acala/eth-rpc-adapter:0a99a2f
```
Comment (shunjizhan) [inline docker-compose-example.yml:75]:
```suggestion
    image: acala/eth-rpc-adapter:v2.5.22
```

**Pull request merged**